pytensor/scalar/basic.py

```diff
@@ -4001,7 +4144,7 @@ class Composite(ScalarOp, HasInnerGraph):
     def __init__(self, inputs, outputs, name="Composite"):
         self.name = name
         # We need to clone the graph as sometimes its nodes already
-        # contain a reference to an fgraph. As we want the Composite
+        # contain a reference to a fgraph. As we want the Composite
```

Comment:
```suggestion
        # contain a reference to an fgraph. As we want the Composite
```
because `fgraph` is spoken _eff_graph